tests/workbookTemplate.test.ts

```diff
@@ -0,0 +1,52 @@
+import workbookTemplate from "../src/workbookTemplate";
+import { pqUtils } from "../src/utils";
+import { pqCustomXmlPath } from "../src/constants";
+import MashupHandler from "../src/mashupDocumentParser";
+import JSZip from "jszip";
+
+describe("single query template tests", () => {
+    const singleQueryDefaultTemplate =
+        workbookTemplate.SIMPLE_QUERY_WORKBOOK_TEMPLATE;
+    let defaultZipFile;
+
+    beforeAll(async () => {
+        defaultZipFile = await JSZip.loadAsync(singleQueryDefaultTemplate, {
+            base64: true,
+        });
+    });
+
+    test("Default template is valid", async () => {
+        const originalBase64Str = await pqUtils.getBase64(defaultZipFile);
+
+        expect(defaultZipFile).toBeTruthy();
+        expect(originalBase64Str).toBeTruthy();
+    });
+
+    test("ConnectedWorkbook XML exists", async () => {
+        const connectedWorkbookXml = defaultZipFile.file(pqCustomXmlPath);
+
+        expect(connectedWorkbookXml).toBeTruthy();
+    });
+
+    test("Base64 string exists and valid for parsing", async () => {
+        expect(
+            async () => await pqUtils.getBase64(defaultZipFile)
```

Comment:
this is somewhat of a duplicate of a test above, you should probably choose 1 testing approach to use, and use it once.